plugins/modules/docker_compose_v2.py

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/python
+
+from __future__ import (absolute_import, division, print_function)
+from ansible.module_utils.basic import AnsibleModule
+import os
+from python_on_whales.docker_client import DockerClient
+from python_on_whales.exceptions import (
+    DockerException,
+)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: docker_compose_v2
+
+short_description: Manage multi-container Docker applications with Docker Compose v2.
+
+version_added: "1.0.0"
+
+description:
+  - Uses Docker Compose v2 to manage docker services. B(This module requires docker-compose > 2.0.0.)
```

Comment:
Why does 2.0.0 not work? Or do you mean:
```suggestion
  - Uses Docker Compose v2 to manage docker services. B(This module requires docker-compose >= 2.0.0.)
```
?